eth/backend.go

```diff
@@ -483,6 +483,17 @@ func New(ctx context.Context, stack *node.Node, config *ethconfig.Config, logger
 						logItems = append(logItems, eth.ProtocolToString[protocol], strconv.Itoa(count))
 					}
 					logger.Info("[p2p] GoodPeers", logItems...)
+
+					if err := chainKv.Update(context.Background(), func(tx kv.RwTx) error {
```

Comment:
1. you don't need write transaction here: use `chainKv.View` 
2. use existing ctx instead of `context.Background()`